t/plugin/log-rotate.t

```diff
@@ -90,8 +90,8 @@ __DATA__
 --- config
     location /t {
         content_by_lua_block {
-            ngx.log(ngx.WARN, "start xxxxxx")
             ngx.sleep(0.1)
+            ngx.log(ngx.WARN, "start xxxxxx")
```

Comment:
Why does the execution order of ngx.sleep and ngx.log affect the stability of the test?

Reply:
It's obvious. We should ensure the log not be rotated.

Reply:
Got it, thanks for the answer.